integration/fuse/src/main/java/alluxio/fuse/cli/FuseCommand.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.fuse.cli;
+
+import alluxio.AlluxioURI;
+import alluxio.client.file.URIStatus;
+import alluxio.exception.status.InvalidArgumentException;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public interface FuseCommand {
```

Comment:
Please add a interface header javadoc

Reply:
Could you use Command instead of `FuseCommand `? So that you can use CommonUtils#loadCommands instead.